code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -673,6 +673,9 @@ var/global/borer_unlock_types = typesof(/datum/unlockable/borer) - /datum/unlock
 	set desc = "Allows to hide beneath tables or certain items. Toggled on or off."
 	set category = "Alien"
 
+	if(stat != CONSCIOUS)
```

Comment:
Also checks for zombiepowder although in this case I guess it doesn't matter, but it's good for OOP in case something is later added to borers so you can override the isConscious proc to check for that thing as well